selfservice/flow/continue_with.go

```diff
@@ -0,0 +1,85 @@
+// Copyright © 2023 Ory Corp
+// SPDX-License-Identifier: Apache-2.0
+
+package flow
+
+import "github.com/gofrs/uuid"
+
+// swagger:model continue_with
+type ContinueWith any
+
+// swagger:enum ContinueWithAction
+type ContinueWithAction string
+
+const (
+	ContinueWithActionSetToken       ContinueWithAction = "set_token"
+	ContinueWithActionVerificationUI ContinueWithAction = "verification_ui"
+)
+
+var _ ContinueWith = new(ContinueWithSetToken)
+
+// Indicates that a session was issued, and the application should use this token for authenticated requests
+// swagger:model continue_with_set_token
+type ContinueWithSetToken struct {
+	// Action will always be `set_token`
+	//
+	// required: true
+	Action ContinueWithAction `json:"action"`
+
+	// Token is the token of the session
+	//
+	// required: true
+	Token string `json:"token"`
+}
+
+func NewContinueWithSetToken(t string) *ContinueWithSetToken {
+	return &ContinueWithSetToken{
+		Action: ContinueWithActionSetToken,
+		Token:  t,
+	}
+}
+
+var _ ContinueWith = new(ContinueWithVerificationUI)
+
+// Indicates, that the UI flow could be continued by showing a verification ui
+//
+// swagger:model continue_with_verification_ui
+type ContinueWithVerificationUI struct {
+	// Action will always be `verification_ui`
+	//
+	// required: true
+	Action ContinueWithAction `json:"action"`
+	// Flow contains the ID of the verification flow
+	//
+	// required: true
+	Flow struct {
```

Comment:
Please use a dedicated struct for this and give it a `swagger:model` annotation

Reply:
The names get quite long. Would've liked to avoid those.. But fine with me, done.